packer-scripts/macos-image-bootstrap

```diff
@@ -83,14 +101,21 @@ EOF
 }
 
 dot_bashrc_setup() {
-
   echo "--- Overwrite .bashrc with our own."
   cat > ~/.bashrc <<EOF
```

Comment:
You might want to quote `EOF`, as that removes the need to escape the variables in the here file.